source/portable/posix/core_pkcs11_pal.c

```diff
@@ -416,3 +416,35 @@ void PKCS11_PAL_GetObjectValueCleanup( CK_BYTE_PTR pucData,
 }
 
 /*-----------------------------------------------------------*/
+
+CK_RV PKCS11_PAL_DestroyObject( CK_OBJECT_HANDLE xHandle )
+{
+    const char * pcFileName = NULL;
+    CK_BBOOL xIsPrivate = CK_TRUE;
+    CK_RV xResult = CKR_OK;
```

Comment:
maintainability: return status values should always be initialized to failure, e.g. `CK_RV xResult = CKR_OBJECT_HANDLE_INVALID`.

Reply:
Done 👍